.github/actions/setup-go/action.yml

```diff
@@ -1,37 +0,0 @@
-name: "Turborepo Go Setup"
-description: "Sets Go up for CI"
-inputs:
-  github-token:
-    description: "GitHub token. You can pass secrets.GITHUB_TOKEN"
-    required: true
-
-runs:
-  using: "composite"
-  steps:
-    - name: Set up Go
-      uses: actions/setup-go@v3
-      with:
-        go-version: "1.20"
-        cache: true
-        cache-dependency-path: cli/go.sum
```

Comment:
It might make for a smaller/eassier diff if we just removed the "setup Go" & "Setup GRPC protobuf" steps from this action instead of canning the action all together.